modules/twinkleprotect.js

```diff
@@ -718,6 +736,7 @@ Twinkle.protect.protectionWeight = {
 	sysop: 30,
 	templateeditor: 20,
 	flaggedrevs_review: 15,  // Pending Changes level 2 protection alone
+	extendedconfirmed: 12,
```

Comment:
Shouldn't be 12. This is for semiprotected + PC1 (see comment above).  Maybe it is time to increase sysop to 40, TE to 30, then put 30/500 at 20. Unless PC2 alone (15) and semiprotected + PC2 (17) should really be considered "higher" than 30/500?